alpine/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apk update \
  && apk upgrade \
  && apk add --no-cache \
         ca-certificates \
+        msmtp \
```

Comment:
@schildbach these `Dockerfile`s are generated from the `Dockerfile.tmpl.php`, the header comment indicates this. So, these changes should be made in `Dockerfile.tmpl.php` and then populated via `make dockerfile` command.

Reply:
Sorry, wrapping in make is unexpected to me. I usually contain my builds in Docker.

Anyway, pushed a fix for this.